pkg/webhook/webhook.go

```diff
@@ -232,6 +232,7 @@ func (wh *webhook) validatePVCCreateRequest(req *v1beta1.AdmissionRequest) *v1be
 	// get the snapshot object to get snapshotdata object
 	snapObj, err := wh.snapClientSet.OpenebsV1alpha1().VolumeSnapshots(pvc.Namespace).Get(snapname, metav1.GetOptions{})
 	if err != nil {
+		glog.Errorf("Error retrieving snapshot name error: '%v'", err)
```

Comment:
@mittachaitu .. think of debugging if any issue comes. Will these log messages helps us?
All it gives is - lot of following logs:
Error retrieving snapshot name error: abcd
Add more details to all error messages

Reply:
I will update it with relevant information.